auth-web/src/routes/router.ts

```diff
@@ -849,6 +851,21 @@ export function getRoutes (): RouteConfig[] {
         allowedRoles: [Role.CreateCredits, Role.FasRefund]
       }
     },
+    {
+      path: '/staff/involuntary-dissolution',
+      name: 'involuntary-dissolution',
+      component: InvoluntaryDissolution,
+      props: true,
+      meta: {
+        allowedRoles: [Role.Staff],
```

Comment:
This is a staff only screen. So, it's going to have only staff as the allowed roles.